clients/destination.go

```diff
@@ -88,6 +88,7 @@ func NewDestinationClient(ctx context.Context, registry specs.Registry, path str
 		}
 		org, name := pathSplit[0], pathSplit[1]
 		localPath := filepath.Join(c.directory, "plugins", string(PluginTypeDestination), org, name, version, "plugin")
+		localPath = withBinarySuffix(localPath)
```

Comment:
I don't think we should save it with `.exe` (I get the goreleaser part but no need to do it on the save part as it will work without the exe)

Reply:
See my comment, it doesn't work without it as the `go` code looks for the extension:
https://github.com/golang/go/blob/f6d844510d5f1e3b3098eba255d9b633d45eac3b/src/os/exec/lp_windows.go#L39

See failure https://github.com/cloudquery/cloudquery/actions/runs/3173117211/jobs/5168369062#step:7:9
![image](https://user-images.githubusercontent.com/26760571/193607796-43da0ed7-d34a-43b7-bbac-5b3022abe3bb.png)